test/validationTests.js

```diff
@@ -9,7 +9,7 @@ describe('Postal codes validation: ', function () {
     Object.keys(countriesData).map(function (alpha2Code) {
 
         var formatFileName = countriesData[alpha2Code].postalCodeFormat;
-        if ( !formatFileName ) {
+        if (!formatFileName) {
```

Comment:
Would it be possible to leave the space around brackets.

Reply:
Sure, should we write down a style guide so that contributors know to leave spaces around brackets?

Reply:
If you'd like to contribute such, that would be awesome ;) 

I think the 'open source world' (possibly unwritten) convention is to try follow what's been used.